DESCRIPTION

```diff
@@ -28,17 +28,15 @@ Imports:
     checkmate,
     cli,
     dplyr (>= 1.0.2),
-    glue (>= 1.4.2),
     haven (>= 2.5.0),
-    janitor,
```

Comment:
We still rely on using `janitor::make_clean_names` and `tm::stemDocument()` inside the `xportr_tidy_rename()`. So, I think we cannot remove them yet.

Reply:
I did the same thing.  The function is in `dev` folder and not used in xportr.

Reply:
🙈 Sorry didn't realize the search hit was inside the `dev` folder.

Reply:
I said the same thing to the person at GSK reviewing the content lol